tests/integration/GKE.md

```diff
@@ -68,6 +68,7 @@ gcloud container clusters \
  - `CLUSTER_NAME`: Whatever suits your fancy, 'istio-e2e' is a good choice.
  - `ZONE`: 'us-central1-f' is a good value to use.
  - `PROJECT_NAME`: is the name of the GCP project that will house the cluster. You get a project by visiting [GCP](https://console.cloud.google.com).
+ - `PROJECT_ID`: is the ID of the GCP project that will house the cluster. You get a project by visiting [GCP](https://console.cloud.google.com).
```

Comment:
Are you replacing PROJECT_NAME? If so, delete the old value.

Also, you need to update the description above.

Reply:
@nmittler The description says ID instead of name, is there any other changes you had in mind that you want me to add?